docker/rholang-web/Dockerfile

```diff
@@ -34,6 +34,9 @@ RUN python3 manage.py migrate
 
 COPY docker/rholang-web/run-server .
 
+RUN python3 -c \
+    "from urllib.request import urlretrieve as wget; wget('https://github.com/django/django/raw/master/LICENSE', filename='/opt/rholangweb/rholang/main/static/django-license')"
```

Comment:
```
ADD https://github.com/django/django/raw/master/LICENSE /opt/rholangweb/rholang/main/static/django-license
```
works fine.

Reply:
ADD causes `docker build` to generate a different image hash each time. So I searched around and found:

> Because image size matters, using ADD to fetch packages from remote URLs is strongly discouraged; you should use curl or wget instead.
 -- https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Silly me didn't include results of my research on this in my commit message...

Reply:
`pip3 install -r requirements.txt` installs Django which I believe includes the LICENSE with it (as part of the source). Is this to make the LICENSE more explicit?

Reply:
I checked, and not only does `pip install django` not include their LICENSE, the name of the copyright holder, **django foundation** occurs *nowhere* in the files that get installed.

Reply:
I'm not sure how I got python3.5/site-packages/Django-1.11.dist-info/LICENSE.txt on my computer but I'm assuming it somehow came with pip install . It shouldn't change with Docker right?

Reply:
_dons dunce cap_

1. I only looked in `site-packages/django` and not `site-packages/Django-1.11.dist-info`
2. The name of the copyright holder is **Django Software Foundation**, not Django Foundation.